components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```
@@ -520,7 +522,15 @@ def main(argv):
     logger.info(f"Starting {COMPRESSION_SCHEDULER_COMPONENT_NAME}")
     sql_adapter = SqlAdapter(clp_config.database)
 
-    task_manager = CeleryTaskManager()
+    if clp_config.compression_scheduler.type == OrchestrationType.celery:
+        task_manager = CeleryTaskManager()
+    elif clp_config.compression_scheduler.type == OrchestrationType.spider:
+        task_manager = SpiderTaskManager(clp_config.spider_db.get_url())
+    else:
+        logger.error(
+            f"Unsupported compression scheduler type:" f" {clp_config.compression_scheduler.type}"
```

Ruff (0.14.4)

Implicitly concatenated string literals on one line

Combine string literals

(ISC001)